sdk/nodejs/rome.json

```diff
@@ -20,10 +20,19 @@
             "recommended": false,
             "complexity": {
                 "noExtraBooleanCast": "error",
+                "useFlatMap": "error",
                 "useOptionalChain": "error"
             },
             "performance": {
                 "noDelete": "error"
+            },
+            "suspicious": {
+                "noLabelVar": "error",
+                "noExplicitAny": "off",
+                "noAsyncPromiseExecutor": "error",
+                "noEmptyInterface": "off",
+                "noShadowRestrictedNames": "off",
+                "noExtraNonNullAssertion": "off"
```

Comment:
One troublesome bug in Rome that I've noticed is that turning on certain lints enables the entire category, so you have to disable other lints in that category explicitly. :(